webgpu.h

```diff
@@ -843,6 +838,8 @@ typedef void (*WGPUProcAdapterRequestDevice)(WGPUAdapter adapter, WGPUDeviceDesc
 
 // Procs of Buffer
 typedef void (*WGPUProcBufferDestroy)(WGPUBuffer buffer);
+typedef void * const (*WGPUProcBufferGetConstMappedRange)(WGPUBuffer buffer);
```

Comment:
I think this should be:
```suggestion
typedef void const * (*WGPUProcBufferGetConstMappedRange)(WGPUBuffer buffer);
```